microsoft365defender_connector.py

```diff
@@ -1190,6 +1350,11 @@ def initialize(self):
         self._tenant = config[DEFENDER_CONFIG_TENANT_ID]
         self._client_id = config[DEFENDER_CONFIG_CLIENT_ID]
         self._client_secret = config[DEFENDER_CONFIG_CLIENT_SECRET]
+        self._timeout = config.get('timeout', DEFAULT_TIMEOUT)
```

Comment:
Can we move "timeout" key to the consts to make it consistent with the rest of the keys above?

Reply:
Done